boot/modeenv.go

```diff
@@ -38,6 +38,10 @@ import (
 
 type bootAssetsMap map[string][]string
 
+// bootCommandLines is a list of kernel command lines. The command lines are
+// marshalled as JSON as a comma can be present in the module parameters.
+type bootCommandLines []string
```

Comment:
just to be clear, this is a list since we could have the kernel command line change when i.e. a new gadget snap is updated which changes the kernel command line right? In normal circumstances this will only have one element in it, but during updates it could have two?

Reply:
Yes, the same approach as other 'lists' in the modeenv, we'll append the new one at the end and then drop the old one after the boot is successful.

Reply:
thanks for confirming